odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/reactive/ReactiveEnumValueRepository.java

```diff
@@ -0,0 +1,11 @@
+package org.opendatadiscovery.oddplatform.repository.reactive;
+
+import java.util.List;
+import org.opendatadiscovery.oddplatform.model.tables.pojos.EnumValuePojo;
+import reactor.core.publisher.Flux;
+
+public interface ReactiveEnumValueRepository extends ReactiveCRUDRepository<EnumValuePojo> {
+    Flux<EnumValuePojo> getEnumValuesByFieldId(final long datasetFieldId);
```

Comment:
this is not required, but why have you truncated datasetField to field? :)

Reply:
Can you elaborate? I don't think I follow your comment

Reply:
rename `getEnumValuesByFieldId` to `getEnumValuesByDatasetFieldId`

Reply:
If you mean why this method is `getEnumValuesByFieldId` instead of `getEnumValuesByDatasetFieldId` - this is simply because how it was named in old repository, and since I didn't change the logic of this method (and also of a corresponding service method), I just kept it the same. But I think it makes sense to rename it to the latter version.